src/extra/widgets/animbtn/lv_animbtn.h

```diff
@@ -30,14 +30,17 @@ typedef enum {
     _LV_ANIMBTN_STATE_NUM,
 } lv_animbtn_state_t;
 
-/**********************
- *      TYPEDEFS
- **********************/
+typedef enum {
+    LV_ANIMBTN_CTRL_FORWARD  = 0,
+    LV_ANIMBTN_CTRL_BACKWARD = 1,
+    LV_ANIMBTN_CTRL_LOOP     = 8,
```

Comment:
Why don't we use the values from `lv_rlottie_ctrl_t`, like `LV_RLOTTIE_CTRL_LOOP`?

Reply:
Sure, let's do that. 
I'm not using `lv_rlottie_ctrl_t` since not all value are valid here (only forward, backward and loop)